src/System.Private.ServiceModel/tools/CertificateGenerator/CertificateGenerator.csproj

```diff
@@ -1,36 +1,14 @@
-﻿<?xml version="1.0" encoding="utf-8"?>
-<Project ToolsVersion="14.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
   <PropertyGroup>
     <OutputType>Exe</OutputType>
-    <RootNamespace>CertificateGenerator</RootNamespace>
-    <AssemblyName>CertificateGenerator</AssemblyName>
-    <TargetFrameworkVersion>v4.5</TargetFrameworkVersion>
+    <TargetFramework>net471</TargetFramework>
   </PropertyGroup>
 
   <ItemGroup>
     <PackageReference Include="BouncyCastle.Crypto.dll" Version="1.8.1" GeneratePathProperty="true" />
     <PackageReference Include="Microsoft.NETFramework.ReferenceAssemblies" Version="1.0.2" PrivateAssets="All" />
```

Comment:
You don't need this package anymore. It's automatically included.

Reply:
I tested it offline, the result shows that without referencing package `Microsoft.NETFramework.ReferenceAssemblies`, the below build command got "_error MSB3644: The reference assemblies for .NETFramework,Version=v4.7.1 were not found._"  
Did I miss something?

`eng\common\cibuild.cmd -warnAsError 0 -configuration Release -projects src\System.Private.ServiceModel\tools\CertificateGenerator\CertificateGenerator.sln /p:Sign=false`

Reply:
I'll take a look. Don't treat this feedback as blocking meanwhile.

Reply:
@imcarolwang I noticed that you introduced a PackageReference to ConfigurationManager which isn't necessary as `System.Configuration` is already inbox in .NET Framework and can just be directly referenced. The ConfigurationManager package is just a full facade that type forwards to System.Configuration. I pushed to your branch to clean that up. I hope that's OK.

Regarding this issue, I couldn't reproduce the error. Can you please try this again?

Reply:
@ViktorHofer Thank you for your help! I pull the code and retry, it works perfectly. I have no idea why it failed consistently last time. Now it works consistently :) Thanks again!